static/app/views/ddm/scratchpad.tsx

```diff
@@ -46,6 +46,7 @@ export function MetricScratchpad() {
           index={index}
           onSelect={setSelectedWidgetIndex}
           isSelected={selectedWidgetIndex === index}
+          numberOfSiblings={widgets.length - 1}
```

Comment:
nit: Could change `isSelected` to `isHighlighted` so we don't need another prop.
```suggestion
          isHighlighted={selectedWidgetIndex === index && widgets.length > 1}
```

Reply:
Was thinking the same thing, but we need isSelected separated from border highlighting for the string representation.

Reply:
k, did not see that 👍